test/e2e/kubectl.go

```diff
@@ -541,24 +541,48 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 
 			By("Waiting for Redis master to start.")
 			waitForOrFailWithDebug(1)
+
 			// Pod
-			forEachPod(func(pod api.Pod) {
-				output := framework.RunKubectlOrDie("describe", "pod", pod.Name, nsFlag)
-				requiredStrings := [][]string{
-					{"Name:", "redis-master-"},
-					{"Namespace:", ns},
-					{"Node:"},
-					{"Labels:", "app=redis"},
-					{"role=master"},
-					{"Status:", "Running"},
-					{"IP:"},
-					{"Controllers:", "ReplicationController/redis-master"},
-					{"Image:", redisImage},
-					{"State:", "Running"},
-					{"QoS Class:", "BestEffort"},
-				}
-				checkOutput(output, requiredStrings)
-			})
+			// this is terrible but we want 1.4.0 alpha to count as well
+			classVersion, err := framework.KubectlVersionGTE(version.MustParse("v1.4.0-alpha"))
```

Comment:
I'm not super familiar with this code.  If this doesn't require updates against in 1.5, then its ok with me.  Otherwise, I'm actually find with just removing the QoS class as a `requiredString` entirely.